website/www/site/layouts/partials/header.html

```diff
@@ -57,7 +57,7 @@
           <a class="navbar-link" href={{ "/about" | relLangURL }}>{{ T "nav-about" }}</a>
         </li>
         <li>
-         <a class="navbar-link" href={{ "/get-started/beam-overview/" | relLangURL }}>{{ T "nav-get-started" }}</a>
+         <a class="navbar-link" href={{ "/get-started/" | relLangURL }}>{{ T "nav-get-started" }}</a>
```

Comment:
It looks like you just changed this for the mobile nav (or something else?). This didn't seem to fix the main desktop version - http://apache-beam-website-pull-requests.storage.googleapis.com/23490/index.html - it looks like there's another reference that you can update on line 19 - https://github.com/apache/beam/blob/3248486a25f905c438b8dd41282bddc71410b467/website/www/site/layouts/partials/header.html#L19

Reply:
Turns out it was line 128 that made the difference on the desktop version.